webclient/src/components/CardDetails/CardDetails.tsx

```diff
@@ -0,0 +1,128 @@
+                )
+              }
+
+              {
+                !card.manaCost ? null : (
+                  <div className='cardDetails-attribute'>
+                    <span className='cardDetails-attribute__label'>Cost:</span>
+                    <span className='cardDetails-attribute__value'>{card.manaCost.replace(/\{|\}/g, '')}</span>
+                  </div>
+                )
+              }
+
+              {
+                !card.convertedManaCost ? null : (
+                  <div className='cardDetails-attribute'>
+                    <span className='cardDetails-attribute__label'>CMC:</span>
+                    <span className='cardDetails-attribute__value'>{card.convertedManaCost}</span>
+                  </div>
+                )
+              }
+
+              {
+                !card.colorIdentity?.length ? null : (
+                  <div className='cardDetails-attribute'>
+                    <span className='cardDetails-attribute__label'>Identity:</span>
+                    <span className='cardDetails-attribute__value'>{card.colorIdentity.join('')}</span>
+                  </div>
+                )
+              }
+
+              {
+                !card.colors?.length ? null : (
+                  <div className='cardDetails-attribute'>
+                    <span className='cardDetails-attribute__label'>Color(s):</span>
+                    <span className='cardDetails-attribute__value'>{card.colors.join('')}</span>
+                  </div>
+                )
+              }
+
+              {
+                !card.types?.length ? null : (
+                  <div className='cardDetails-attribute'>
+                    <span className='cardDetails-attribute__label'>Main Type:</span>
+                    <span className='cardDetails-attribute__value'>{card.types.join(', ')}</span>
+                  </div>
+                )
+              }
+
+              {
+                !card.type ? null : (
+                  <div className='cardDetails-attribute'>
+                    <span className='cardDetails-attribute__label'>Type:</span>
+                    <span className='cardDetails-attribute__value'>{card.type}</span>
+                  </div>
+                )
+              }
+
+              {
+                !card.side ? null : (
+                  <div className='cardDetails-attribute'>
+                    <span className='cardDetails-attribute__label'>Side:</span>
+                    <span className='cardDetails-attribute__value'>{card.side}</span>
+                  </div>
+                )
+              }
+
+              {
+                !card.layout ? null : (
+                  <div className='cardDetails-attribute'>
+                    <span className='cardDetails-attribute__label'>Layout:</span>
+                    <span className='cardDetails-attribute__value'>{card.layout}</span>
+                  </div>
+                )
+              }
+            </div>
```

Comment:
Missing Loyalty, Hand, etc. You can grab them all from  https://github.com/Cockatrice/Cockatrice/blob/141de4628e56f43f0d8d2b73f0589ce2928ec46c/cockatrice/src/game_specific_terms.h#L26